promtail/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM grafana/promtail:main
```

Comment:
Should we pin a version here?

Reply:
Used the version specified in: https://artifacthub.io/packages/helm/grafana/promtail (AppVersion: 3.0.0)